doc/getting-started/apl-nuc.rst

```diff
@@ -61,11 +61,20 @@ complete this setup.
 
       # swupd autoupdate --disable
 
+#. if an old Clear Linux version installed already on your Hardware. We recommend that you upgrade Clear Linux to 23690 
```

Comment:
Change to:

    #. If you have an older version of Clear Linux already installed
        on your hardware, use this command to upgrade Clear Linux
        to version 23690 (or newer):